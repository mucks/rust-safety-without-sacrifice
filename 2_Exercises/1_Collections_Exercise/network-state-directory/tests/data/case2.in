# Move success and failure
MEMBER ForestCity 1 Alice Citizen
MEMBER Prospera 7 Paula Validator
MOVE ForestCity Prospera 1
MOVE ForestCity Prospera 99
COUNT ForestCity
COUNT Prospera
STATS
