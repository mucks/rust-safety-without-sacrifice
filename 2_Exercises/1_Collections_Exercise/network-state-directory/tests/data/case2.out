Moved 1 from ForestCity to Prospera: true
Moved 99 from ForestCity to Prospera: false
COUNT ForestCity: 0
COUNT Prospera: 2
STATS citizens=1 validators=1 founders=0
